.github/workflows/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,42 @@
+name: Bug Report
+description: Create a bug report for RIOT-rs
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for filing a bug report!
+  - type: textarea
+    attributes:
+      label: Summary
+      description: Please provide a short summary of the bug, along with any information you feel relevant to replicate the bug.
+    validations:
+      required: true
```

Comment:
To me it makes sense to split this into a question asking for expected behaviour and actual behaviour.

If you want to be exhaustive we could even ask for a minimal example/steps to reproduce the issue.

What do you think?

Reply:
I initially had it in the template, but then decided to remove it. From my own experience, "expected behavior" and "actual behavior" often had a large overlap with the summary section. But I am okay with adding it.

> If you want to be exhaustive we could even ask for a minimal example/steps to reproduce the issue.

:+1: I'm in favor of that.

Reply:
> I initially had it in the template, but then decided to remove it. From my own experience, "expected behavior" and "actual behavior" often had a large overlap with the summary section. But I am okay with adding it.

Fine with leaving it out with me. We can always add it later when we notice it gets omitted in initial reports.

Reply:
(Note to myself: I should stop writing comments and then directly editing them, because people seems to faster at replying then I am at my edit :D)